.github/workflows/webpack.yml

```diff
@@ -0,0 +1,24 @@
+name: sentryConfig
+
+on:
+  release:
+    types: [published, prereleased]
+
+
+jobs:
+  createSentryRelease:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - name: Create a Sentry.io release
+        uses: tclindner/sentry-releases-action@v1.0.0
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+          SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
+        with:
+          tagName: ${{ github.ref }}
+          environment: master
+          releaseNamePrefix: Rhel-Advisor
```

Comment:
Maybe we better use the prefixes that are up-to-date with chrome-service-backend? Like in our case `advisor` https://github.com/RedHatInsights/chrome-service-backend/blob/main/static/stable/prod/modules/fed-modules.json#L224

Reply:
im not opposed to this, I mainly just wanted to differenciate ocp-advisor and rhel-advisor down the line